frame/examples/dev-mode/src/lib.rs

```diff
@@ -60,7 +60,7 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
-		#[pallet::call_index(0)]
+		/// No need to define a `call_index` attribute here because of `dev_mode`.
 		/// No need to define a `weight` attribute here because of `dev_mode`.
```

Comment:
```suggestion
		// No need to define a `call_index` attribute here because of `dev_mode`.
		// No need to define a `weight` attribute here because of `dev_mode`.
```
Its not really a doc thing, rather just a code comment.